README.md

```diff
@@ -26,5 +26,5 @@ cd paradise-papers-django
 cd paradise_papers_django
```

Comment:
change both `cd paradise_papers_django` to `cd paradise_papers_search`

and add to Requirements
`django-neomodel 0.0.4`